python/pip_install/pip_repository.bzl

```diff
@@ -111,6 +112,18 @@ def _parse_optional_attrs(rctx, args):
 
     return args
 
+def _detect_changes(rctx):
```

Comment:
I don't think this is strictly necessary. Just having a label list of the source files in the attrs of the repo rule will add the labels to the repository marker file.